src/homepageExperience/utils.ts

```diff
@@ -6,27 +6,27 @@ export const HOMEPAGE_NAVIGATION_STEPS = [
     glyph: IconFont.BookOutline,
   },
   {
-    name: 'Install\nDependencies',
+    name: `Install\n Dependencies`,
     glyph: IconFont.Install,
   },
   {
     name: 'Tokens',
     glyph: IconFont.CopperCoin,
   },
   {
-    name: 'Initialize\nClient',
+    name: 'Initialize\n Client',
```

Comment:
Why the space? I think we decided that not having a space was preferred here, since these are separated by a new line. Having a trailing or leading space, while not the end of the world, might cause issues for screen readers and will show up when copy/pasting highighted words. It might also affect string literals when searching or in e2e tests

Reply:
So when copying and pasting without the space, it pasted as `InstallDependencies` with no space. But if that space will mess with screen readers, it probably is better to remove the space since I doubt users will actually copy and paste those nav items often if at all.

Reply:
This is a very minor nitpick, and if it copy/pasted wrong, then it's fine to leave it as it is :)